src/components/buttons/button-accessibility-image-alt.jsx

```diff
@@ -27,10 +30,10 @@ class ButtonAccessibilityImageAlt extends React.Component {
 		editor: PropTypes.object.isRequired,
 	};
 
-	constructor(prop, context) {
-		super(prop, context);
+	constructor(prop) {
+		super(prop);
```

Comment:
Constructors don't receive `context` (that was part of the legacy context API), and in any case, I don't think we were ever using it here.